### PR TITLE
feat: first draft for cookie consent banner

### LINKED ISSUE #765: Cookie Consent
### Describe the chore

Create a cookie banner for `lib`. Needed for `landing-page` and `tasks`

### PR DESCRIPTION
**Pull request by florian-sabonchi**
## Which issues does this pull request close?
closes #765

## [OPTIONAL] 
marked as draft because we need to check which cookies are used

### CODE REVIEW

**Review by use-to — CHANGES_REQUESTED** [review]
image

**Comment by florian-sabonchi** [comment]
> image

Please not as screenshot

**Comment by use-to** [comment]
> > image
> 
> Please not as screenshot

What do you mean? Do you need further explanation?

The links at the bottom of our footer should be removed. Instead, try to integrate it in to the footer section. (`General`)

File: landing-page/components/CookieConsentConfig.tsx
Comment (MaxSchaefer) [inline landing-page/components/CookieConsentConfig.tsx:1]: This file does not contain a component. Utils or `CookieConsent.tsx`?

**Comment by MaxSchaefer** [comment]
@florian-sabonchi as already mentioned, please add the cookie settings into the general section of the footer.

File: landing-page/components/CookieConsentConfig.tsx
Comment (MaxSchaefer) [reply landing-page/components/CookieConsentConfig.tsx:1]: @florian-sabonchi this is still no component.

**Review by use-to — CHANGES_REQUESTED** [review]

**Review by use-to — APPROVED** [review]
Squash, please.

**Comment by use-to** [comment]
@MaxSchaefer Any concerns?

**Review by MaxSchaefer — APPROVED** [review]
Very nice!

In the future, we need to add AnchorFM (Podcast) and Cloudflare (cdn.helpwave.de) to our privay policy as well.

**Comment by use-to** [comment]
> Very nice!
> 
> In the future, we need to add AnchorFM (Podcast) and Cloudflare (cdn.helpwave.de) to our privay policy as well.

Already did in https://cdn.helpwave.de/privacy.html
We don't have to add it to the cookie banner in this case.

**Pull request merged**